mne/preprocessing/ica.py

```diff
@@ -786,7 +786,7 @@ def _export_info(self, info, container, add_channels):
         ch_names = []
         ch_info = info['chs'] = []
         for ii in range(self.n_components_):
-            this_source = 'ICA %03d' % (ii + 1)
+            this_source = 'ICA %03d' % (ii)
```

Comment:
channel names for ICA are called ICA 001, ICA 002 etc.

if you want to display as number it should be displayed as ICA #0, ICA #1 etc.